bluesky/plan_stubs.py

```diff
@@ -743,6 +748,12 @@ def inner_trigger_and_read():
             if hasattr(obj, 'trigger'):
                 no_wait = False
                 yield from trigger(obj, group=grp)
+        yield from create(streaming_name)
+        for obj in devices:
+            reading = (yield from read_streaming(obj))
+            if reading is not None:
+                ret.update(reading)
```

Comment:
I guess there is no point in capturing these results because we capture redundant results below. If these reading are different than the later readings something is broken. Perhaps, if anything, we should collect these results separately and asset that the subsequent readings are equal.

Reply:
yes you're right, these are returned results. i copied and pasted without looking in depth... :-|